modules/tinymce/src/themes/silver/test/ts/browser/editor/SilverFixedToolbarContainerPriorityTest.ts

```diff
@@ -0,0 +1,38 @@
+﻿import { Assertions } from '@ephox/agar';
+import { after, before, describe, it } from '@ephox/bedrock-client';
+import { TinyHooks, TinyUiActions } from '@ephox/mcagar';
+import { Insert, Remove, SugarBody, SugarElement } from '@ephox/sugar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Theme from 'tinymce/themes/silver/Theme';
+
+describe('browser.tinymce.themes.silver.editor.SilverFixedToolbarContainerPriorityTest', () => {
+  const toolbar: SugarElement<HTMLDivElement> = SugarElement.fromHtml('<div style="margin: 50px 0;"></div>');
+  const toolbarWithId: SugarElement<HTMLDivElement> = SugarElement.fromHtml('<div id="toolbar" style="margin: 50px 0;"></div>');
+  before(() => {
+    Insert.append(SugarBody.body(), toolbar);
+    Insert.append(SugarBody.body(), toolbarWithId);
+  });
+
+  after(() => {
+    Remove.remove(toolbar);
```

Comment:
I think `toolbarWithId` needs to be removed here as well

Reply:
Yeah indeed. Nice find!